content/en/docs/installation/upgrading/_index.md

```diff
@@ -69,5 +78,37 @@ begin the upgrade process like so - replacing `<version>` with the version
 number you want to install:
 
 ```bash
-$ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/<version>/cert-manager.yaml
+$ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/<version>/cert-manager.yaml
 ```
+
+Once you have deployed the new version of cert-manager, you can [verify](../verify/) the installation.
+
+## Reinstalling cert-manager
+
+In some cases there may be a need to do a full uninstall and re-install of
+cert-manager. An example could be when a very old cert-manager version needs to
+be brought up to date and it's not feasible to upgrade one minor version at a
+time, which is our default recommended upgrade strategy.
+
+If cert-manager `CustomResourceDefinition`s are also uninstalled, this will mean
+loss of associated cert-manager custom resources such as `Certificate`s. The
+main concern associated with this is application downtime and unnecessary
+certificate reissuance, that could happen if `Secret`s with the X.509
+certificates get deleted. You can use [`--enable-certificate-owner-ref`
+flag](https://github.com/cert-manager/cert-manager/blob/f225637e200a864948695cca19620aeed6428157/cmd/controller/app/options/options.go#L339-L341)
+on cert-manager controller to configure whether the `Secret`s should be deleted.
+If this flag is set to true, each `Secret` will have owner reference to the
+`Certificate` for which it was created and when the `Certificate` is deleted,
+the `Secret` will be garbage collected. The default value for this flag is
```

Comment:
Not a fan of `garbage collected` as the description. I'm pretty sure they will be *actively* deleted, not simply "eventually discarded at some random future point".